.travis.yml

```diff
@@ -0,0 +1,89 @@
+# Use trusty for better performance (and avoiding mysql/postgres/solr gone issues on precise and container infra)
+dist: trusty
+sudo: required
+
+language: php
+
+services:
+  - mysql
+
+# Mysql isn't installed on trusty (only client is), so we need to specifically install it
+addons:
+  apt:
+    packages:
+    - mysql-server-5.6
+    - mysql-client-core-5.6
+    - mysql-client-5.6
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+env:
+  global:
+    # For functional and acceptance tests
+    - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
+    # In order to specify extra flags like ignoring platform reqs, using only stable packages and so on.
+    - COMPOSER_FLAGS=""
```

Comment:
Can be removed?

Reply:
I would leave it here just for sake of consistency and future options